packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -49,6 +49,18 @@
             </component>
             <!-- END ERROR BOUNDARY -->
 
+            <!-- Local Legend label -->
+            <!-- START ERROR BOUNDARY -->
+            <component
+                :is="errorBoundary"
+                tier="3">
+                <div
+                    v-if="isPremier"
+                    :class="[$style['c-restaurantCard-localLegend']]">
+                    <legend-icon data-test-id="local-legend-icon" />
```

Comment:
can we rename this to something like `premier-restaurant-icon`? In case the local legend thing changes. We probably shouldn't reference local legends directly and just refer to it as a premier restaurant.

Reply:
same for css classes